NMDSdevopsServiceAdm/DataEngineering
# Add location id hash

# Description
Add a hashed location ID to the cqc ratings for data requests

# How to test
Unit tests passing
Run in branch: https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/add-location-id-hash-flatten_cqc_ratings_job/runs
Column appears in athena

# Developer checklist
- [x] Linked to Trello ticket: https://trello.com/c/pyxv5axJ/839-add-hashed-identifier-to-the-cqc-ratings-file
